src/containers/guest-home-page/sign-up-form/SignUpForm.tsx

```diff
@@ -0,0 +1,161 @@
+import Box from '@mui/material/Box'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import { useTranslation } from 'react-i18next'
+import { styles } from '~/containers/guest-home-page/sign-up-form/SignUpForm.styles'
+import useInputVisibility from '~/hooks/use-input-visibility'
+import React, { useCallback, useState } from 'react'
+import { SignupParams } from '~/types'
+import Typography from '@mui/material/Typography'
+import ButtonBase from '@mui/material/ButtonBase'
+import AppButton from '~/components/app-button/AppButton'
+import { useAppSelector } from '~/hooks/use-redux'
+import Checkbox from '@mui/material/Checkbox'
+import FormControlLabel from '@mui/material/FormControlLabel'
```

Comment:
Try to reorder imports. First imports from react and other libs, after that our own hooks and components.